app/models/manageiq/providers/automation_manager.rb

```diff
@@ -5,6 +5,7 @@ class ManageIQ::Providers::AutomationManager < ::ExtManagementSystem
   has_many :configured_systems,           :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_profiles,       :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_scripts,        :dependent => :destroy, :foreign_key => "manager_id"
+  has_many :configuration_script_authentications, :through => :configuration_scripts, :source => :authentications
```

Comment:
@Ladas I dont think we'll need `:dependent => :destroy` here, right?
Because its already [here](https://github.com/durandom/manageiq/blob/5287aaf56c29d62c4d989b12553716ccd33c4720/app/models/configuration_script_base.rb#L14-L14)